enterprise/core-edge/src/main/java/org/neo4j/coreedge/core/EnterpriseCoreEditionModule.java

```diff
@@ -121,6 +126,18 @@ public void registerProcedures( Procedures procedures )
         }
     }
 
+    @Override
+    protected Log authManagerLog( Config config, FileSystemAbstraction fileSystem, JobScheduler jobScheduler )
+            throws IOException
+    {
+        if (securityLog == null)
+        {
+            securityLog = new SecurityLog( config, fileSystem,
+                    jobScheduler.executor( JobScheduler.Groups.internalLogRotation ) );
+        }
```

Comment:
I tried this, but I don't see how it's possible, since there isn't a common superclass with access to the enterprise `SecurityLog` that can access the field.